.buildkite/filebeat/scripts/unit-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/env-scripts/linux-env.sh
+source .buildkite/env-scripts/macos-env.sh
+
+echo ":: Execute Unit Tests ::"
+sudo chmod -R go-w filebeat/
+
+cd filebeat
```

Comment:
`cd filebeat`  is a good solution but it might be error prone because it changes the dir
I'd recommend using `mage -d filebeat build unitTest` instead. It does the same but without `cd`

Reply:
It worked with unit tests, but same approach caused [error](https://buildkite.com/elastic/filebeat/builds/2273#_) in integration tests (both go & python). If it's ok, i'd keep the `cd filebeat` for meantime and explore the issue later, once we will unify the scripts for all the beats.